products/iam/api.yaml

```diff
@@ -14,8 +14,8 @@
 # TODO(nelsonjr): Make all Zone and Region resource ref
 
 --- !ruby/object:Api::Product
-name: Google Cloud IAM
-prefix: giam
+name: IAM
```

Comment:
This one is "IAM" where above you have "Dns" - it might be a good idea to standardize on all-caps or first-caps for initialisms.

Reply:
Sure- downcased them all. Might cause some spurious changes, so I deleted the Magician's comment to make sure I catch them all.